mlir/lib/Dialect/Rock/Transforms/BlockwiseGemmToThreadwise.cpp

```diff
@@ -858,14 +872,23 @@ LogicalResult ThreadwiseWriteAllRewritePattern::matchAndRewrite(
   Value bid = b.createOrFold<rock::WorkgroupIdOp>(loc, b.getIndexType());
   Value tid = b.createOrFold<rock::WorkitemIdOp>(loc, b.getIndexType());
 
-  SmallVector<Value, 3> writeStartCoords{tid, zero};
-  SmallVector<int64_t, 3> bounds{1, iterLen};
-  SmallVector<int64_t, 3> strides{1, vectorLen};
+  SmallVector<Value, 3> writeStartCoords;
+  SmallVector<int64_t, 3> bounds;
+  SmallVector<int64_t, 3> strides;
+  ValueRange extraIndices = op.getExtraIndices();
+  for (Value extraIdx : extraIndices) {
+    writeStartCoords.push_back(extraIdx);
```

Comment:
Nit: There are constructors for both copying in a range and producing a vector of N copies of something.